package.json

```diff
@@ -29,10 +29,10 @@
     "glob": "^7.1.1",
     "loader-utils": "^1.0.2",
     "node-elm-compiler": "^4.2.1",
-    "yargs": "^6.5.0"
+    "yargs": "^8.0.1"
```

Comment:
I upgraded because it was falling behind but I've reverted the bump to `yargs` and `mocha` :)